src/applications/mhv/medications/tests/e2e/fixtures/listOfPrescriptions.json

```diff
@@ -854,13 +854,13 @@
                 "prescriptionName": "BACITRACIN 500 UNT/GM OPH OINT",
                 "refillStatus": "refillinprocess",
                 "refillSubmitDate": "2023-09-26T20:10:24.000Z",
-                "refillDate": "2023-09-23T04:00:00.000Z",
+                "refillDate": "September 24, 2023",
                 "refillRemaining": 1,
                 "facilityName": "SLC10 TEST LAB",
                 "orderedDate": "2023-08-27T04:00:00.000Z",
                 "quantity": 1,
                 "expirationDate": "2024-08-27T04:00:00.000Z",
-                "dispensedDate": "2023-08-29T04:00:00.000Z",
+                "dispensedDate": "September 24, 2023",
                 "stationNumber": "979",
```

Comment:
what was the reason for the format change?

Reply:
Time does not show in the UI and I used a fixture file that has tracking info for the refill history.

Reply:
Do we actually expect the API to return a date in that format? I'd expect a a consistent format from the backend APIs...

Reply:
Made the suggested changes and pushed the updated one.